package.json

```diff
@@ -11,14 +11,23 @@
     "clean": "npm exec -- tsc --build --clean tsconfig.build.json",
     "publish": "bash scripts/publish",
     "test": "npm run build && jest --verbose --runInBand ./test/functional",
-    "changeset": "changeset"
+    "test:sdk": "npm run build && jest --verbose --runInBand ./test/sdk",
+    "changeset": "changeset",
+    "changeset:version": "changeset version && npm i --package-lock-only",
+    "changeset:publish": "npm run build && changeset publish",
+    "changeset:publish-dev": "npm run build && changeset publish --tag dev",
```

Comment:
You forgot to rebase master ... the below commands do not exist in master branch any more. 

```
  "changeset:version": "changeset version && npm i --package-lock-only",
    "changeset:publish": "npm run build && changeset publish",
    "changeset:publish-dev": "npm run build && changeset publish --tag dev",
```

Reply:
Fixed